float64 t
geometry_msgs/Point position
geometry_msgs/Point velocity
